File: autofin/src/main/java/model/add_lead_details/AddLeadRequest.java
package model.add_lead_details;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AddLeadRequest {
    @SerializedName("UserId")
    @Expose
    private String userId;
    @SerializedName("UserType")
    @Expose
    private String userType;
    @SerializedName("Data")
    @Expose
    private AddLeadDetails data;

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getUserType() {
        return userType;
    }

    public void setUserType(String userType) {
        this.userType = userType;
    }

    public AddLeadDetails getData() {
        return data;
    }

    public void setData(AddLeadDetails data) {
        this.data = data;
    }


}

File: autofin/src/main/java/v2/model/request/AddLeadRequest.kt
package v2.model.dto

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import v2.model.request.add_lead.AddLeadData

@Parcelize
public data class AddLeadRequest(

        @SerializedName("UserId") public var UserId: String? = null,
        @SerializedName("UserType") public var UserType: String? = null,
        @SerializedName("Data") public var Data: AddLeadData? = null


) : Parcelable




